cloudify_cli/commands/snapshots.py

```diff
@@ -62,10 +64,23 @@ def restore(snapshot_id,
         recreate_deployments_envs,
         force,
         tenant_name,
-        restore_certificates
+        restore_certificates,
+        no_reboot
     )
     logger.info("Started workflow execution. The execution's id is {0}".format(
         execution.id))
+    if not restore_certificates:
+        return
+    if no_reboot:
+        logger.warn('Certificates may be replaced during snapshot restore, '
+                    'it is recommended to reboot the Manager VM when the '
+                    'execution is terminated, or else several services may '
+                    'not work!')
```

Comment:
~it~ **It** is recommended ~to~ **that you** reboot the Manager VM when the execution is terminated, or ~else~ several services ~may~ **might** not work ~!~ **.**